build.cake

```diff
@@ -67,22 +68,59 @@ Task("Restore-NuGet-Packages")
     });
 });
 
-Task("Patch-Assembly-Info")
+Task("Run-GitVersion")
     .IsDependentOn("Restore-NuGet-Packages")
+    .IsDependentOn("Run-GitVersion-AppVeyor")
+    .IsDependentOn("Run-GitVersion-Local");
+
+
+Task("Run-GitVersion-AppVeyor")
+    .WithCriteria(AppVeyor.IsRunningOnAppVeyor)
     .Does(() =>
 {
-    var file = "./src/SolutionInfo.cs";
-    CreateAssemblyInfo(file, new AssemblyInfoSettings {
-        Product = "Cake",
-        Version = version,
-        FileVersion = version,
-        InformationalVersion = semVersion,
-        Copyright = "Copyright (c) Patrik Svensson, Mattias Karlsson, Gary Ewan Park and contributors"
+    GitVersion(new GitVersionSettings {
+        UpdateAssemblyInfoFilePath = "./src/SolutionInfo.cs",
```

Comment:
@patriksvensson that is correct.  Gitversion simply applies a regular expression update to the specified file, or it can loop through all files and look for AssemblyInfo.cs files.